### PR TITLE
Feature: Apple login

### PR DESCRIPTION
**Pull request by chrysloi**
# Feature: Apple login

## Description

As a patient I want to be able to login with apple

### Acceptance Criteria

- [x] User with IOS can login with apple

### Steps to test this PR

- clone this repo
- Install project dependencies
- Start the app
- head over the Sign in screen and choose sign in with apple
- follow the steps

⚠️NB: This will only work if you use MacOS and an iPhone or IOS simulator

### CODE REVIEW

File: app.json
Comment (fredshema) [inline app.json:1]: Consider formatting this file

File: app/signupSignin/SignIn.tsx
Comment (fredshema) [inline app/signupSignin/SignIn.tsx:1]: This file too

**Review by fredshema — APPROVED** [review]
Check the formatting of changed files, otherwise the feature is working well on my side

**Review by kibongos40 — APPROVED** [review]
# This works both on new accounts and existing accounts

image
image

File: app.json
Comment (chrysloi) [reply app.json:1]: they're already formatted

**Pull request merged**